puppet/modules/slave/manifests/init.pp

```diff
@@ -101,8 +101,11 @@
     'postgresql-dev':
```

Comment:
Should be use `postgresql::lib::devel` from puppetlabs-postgresql?

Reply:
If it work with SCLs, yes... setting the devel_package_name seems not to suffice..